app/src/main/java/org/p2p/wallet/history/ui/history/HistoryFragment.kt

```diff
@@ -26,11 +27,13 @@ class HistoryFragment :
     }
 
     override val presenter: HistoryContract.Presenter by inject()
+    private val historyListPresenter: HistoryListViewContract.Presenter by inject()
```

Comment:
could you leave a comment on why we need to inject this presenter outside of view?